frontend-service/src/objects/gameConsts.ts

```diff
@@ -0,0 +1,4 @@
+const numberOfRoundsInGame = 5;
+const indexOfFinalRound = numberOfRoundsInGame - 1;
+
+export { numberOfRoundsInGame, indexOfFinalRound };
```

Comment:
Before this was a magic number in every file where the number of rounds was needed. I think it's nicer to separate the index as the it is on an array to the round number as the user sees it beacuse otherwise I would have `indexOfFinalRound -1` scattered around the codebase in a not nice way.